public/app/features/dashboard-scene/utils/dashboardSessionState.ts

```diff
@@ -0,0 +1,114 @@
+import { UrlQueryMap, urlUtil } from '@grafana/data';
+import { config, locationService } from '@grafana/runtime';
+import { UrlSyncManager } from '@grafana/scenes';
+
+import { DashboardScene } from '../scene/DashboardScene';
+
+export function getPreservedSceneURLStateKey() {
+  return `grafana.dashboard.preservedUrlFiltersState`;
+}
```

Comment:
minor nit, this could just be a constant now like `PRESERVED_SCENE_STATE_KEY`

Reply:
yup!